corehq/apps/userreports/templates/userreports/partials/report_builder_configure_report.html

```diff
@@ -54,16 +54,25 @@
                 return that.property() != "";
             });
         };
+        PropertyListItem.wrap = function(o){
+            var i = new PropertyListItem();
+            i.property(o.property);
+            i.displayText(o.display_text);
+            i.format(o.format);
+            return i;
+        };
+
         /**
          * Knockout view model controlling the filter property list.
          */
         var propertyList = function(options) {
             var that = this;
             options = options || {};
+            this.buttonText = options.buttonText != undefined ? options.buttonText : "Add property";
             this.hasFormatCol = ko.observable(options.hasFormatCol != undefined ? options.hasFormatCol : true);
             this.formatOptions = ko.observableArray(["Choice", "Date", "Numeric"]);
             this.propertyOptions = ko.observableArray({{ property_options|JSON }});
-            this.columns = ko.observableArray([]);
+            this.columns = ko.observableArray(options.initialCols != undefined ? options.initialCols : []);
```

Comment:
If you're trying to be explicit here, use `!== undefined`—general rule is basically never use `!=`:

```js
> null == undefined
< true
> null === undefined
< false
```